frontend/appflowy_web_app/env.beta

```diff
@@ -0,0 +1,3 @@
+AF_WS_URL=wss://beta.appflowy.cloud/ws/v1
```

Comment:
should this file name be beta.env?

Reply:
Yes, this is a file used to copy when publishing, not a file used to replace environment variables, which are prohibited from submission.